content/en/blog/2023-06-09-tales-of-open-source-vfscore.md

```diff
@@ -0,0 +1,414 @@
+---
+title: "Tales of Open Source: vfscore"
+date: "2023-06-08T22:00:00+01:00"
+author: "Răzvan Deaconescu"
+tags: ["open source", "vfscore", "Unikraft", "OSv", "filesystem"]
+---
+
+The beauty of open source is the immediate availability of source code that you can use to create or improve your software.
+Of course, this relies on [compatible licensing](https://en.wikipedia.org/wiki/License_compatibility).
+
+Having access to existing, and, ideally, well tested and maintained software, reduces design, development and testing time.
```

Comment:
Possible alternative for writing this paragraph, not necessarily better:
"Access to existing and, ideally, well-tested and maintained software reduces design, development, and testing time while giving software designers and developers the bandwidth to focus on the relevant parts of the software they are creating."

Reply:
Awesome proposal, @Starnox . Thanks.